src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -602,7 +671,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IBinaryKind As BinaryOperationKind Implements IBinaryOperatorExpression.BinaryKind
```

Comment:
Why the odd name for the property here?  Why not call it BinaryKind to match the interface?